### DISCUSSION TITLE
TOLORS but it will be TOLORI

### DISCUSSION THREAD

**Discussion by gazalied** [ScratchAddons/ScratchAddons#1478] [Ideas]
It could be useful for thing like [this](https://github.com/ScratchAddons/ScratchAddons/discussions/1103#discussioncomment-249724).

**Comment by mxmou** [comment]
Implemented? #927

**Comment by gazalied** [comment]
Oh, sorry! How do I close this thing?

**Reply by mxmou** [reply]
Discussions can't be closed the same way as issues can, but you can mark a reply as answer.

**Comment by gazalied** [comment]
Also can you pin the TOLORS?
